### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by baardhermansen**
Trying to change PR from live to master.

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Minor style checks needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/windows/appx/Remove-AppxPackage.md [hunk 1]
```diff
@@ -136,11 +145,13 @@ Accept wildcard characters: False
...
+
 If you specify this parameter, the cmdlet removes the app package for only the user that this cmdlet specifies. To remove a package for a user profile other than the profile of the current user, you must run this command by using administrator permissions.
 
-Note:
-- User "parameter of the "Remove-AppxPackage" command only accepts SID 
-- Use **whoami** command to display the current SID of a user, see [whoami syntax](https://docs.microsoft.com/windows-server/administration/windows-commands/whoami).
+> [!NOTE]
+>
+> - This parameter only accepts user SIDs
+> - Use **whoami** command to display the current SID of a user, see [whoami syntax](https://docs.microsoft.com/windows-server/administration/windows-commands/whoami)
```
Comment (JohanFreelancer9) [inline docset/windows/appx/Remove-AppxPackage.md]:
```suggestion
> - Use the **whoami** command to display the current SID of a user. See [whoami syntax](https://docs.microsoft.com/windows-server/administration/windows-commands/whoami) for more information.

```

File: docset/windows/appx/Remove-AppxPackage.md [hunk 2]
```diff
@@ -136,11 +145,14 @@ Accept wildcard characters: False
...
+
 If you specify this parameter, the cmdlet removes the app package for only the user that this cmdlet specifies. To remove a package for a user profile other than the profile of the current user, you must run this command by using administrator permissions.
 
-Note:
-- User "parameter of the "Remove-AppxPackage" command only accepts SID 
-- Use **whoami** command to display the current SID of a user, see [whoami syntax](https://docs.microsoft.com/windows-server/administration/windows-commands/whoami).
+> [!NOTE]
+>
+> - This parameter only accepts user SIDs
+> - Use the **whoami** command to display the current SID of a user. See [whoami syntax](https://docs.microsoft.com/windows-server/administration/windows-commands/whoami) for details.
```
Comment (get-itips) [inline docset/windows/appx/Remove-AppxPackage.md]:
```suggestion
> - Use the **whoami /user** command to display the current SID of a user. See [whoami syntax](https://docs.microsoft.com/windows-server/administration/windows-commands/whoami) for details.

```

**Pull request merged**